### PR TITLE
Closes #32 Skip parsing of servers property

### LINKED ISSUE #32: OpenAPI spec `servers` property is not accounted for
The `servers` property can be defined at the root level of the spec, or at each endpoint. The latter case is not being accounted for in the parser. In turn there is a parsing error.

```dart
Unhandled exception:
type 'List<dynamic>' is not a subtype of type 'Map<String, dynamic>' in type cast
#0      OpenApiParser.parseRestClients.<anonymous closure>.<anonymous closure> (package:swagger_parser/src/parser/parser.dart:326:13)
```

Sample spec that fails parsing (saved as text file because Github does not allow JSON file uploads): [sample_spec.txt](https://github.com/Carapacik/swagger_parser/files/11368186/index_operations.txt)

`servers` specification: https://swagger.io/docs/specification/api-host-and-base-path/

# Solution

It does not look like `swagger_parser` utilizes or has a need for any of the `servers` properties. This can be ignored during the parsing process.

### PR DESCRIPTION
**Pull request by walsha2**
Resolves:

- #32 
- #35

### CODE REVIEW

File: swagger_parser/lib/src/parser/parser.dart [hunk 1]
```diff
@@ -322,39 +323,41 @@ class OpenApiParser {
...
-        final sameTagIndex =
-            restClients.indexWhere((e) => e.name == currentTag);
-        if (sameTagIndex == -1) {
-          restClients.add(
-            UniversalRestClient(
-              name: currentTag,
-              requests: [request],
-              imports: SplayTreeSet<String>.of(imports),
-            ),
+        if (key != _serversVar) {
```
Comment (walsha2) [inline swagger_parser/lib/src/parser/parser.dart:326]: @Carapacik The line above is really the only change .

File: swagger_parser/lib/src/parser/parser.dart [hunk 2]
```diff
@@ -322,39 +324,48 @@ class OpenApiParser {
...
+              ? parametersV2(requestPath)
+              : parametersV3(requestPath);
+
+          var requestName = (key + path).toCamel;
+          if (requestPath.containsKey(_operationIdVar)) {
+            requestName = requestPath[_operationIdVar].toString().toCamel;
+          }
+
+          final request = UniversalRequest(
+            name: requestName,
```
Comment (walsha2) [inline swagger_parser/lib/src/parser/parser.dart:344]: This part resolves #35

File: swagger_parser/lib/src/parser/parser.dart [hunk 3]
```diff
@@ -322,39 +324,48 @@ class OpenApiParser {
...
-        final sameTagIndex =
-            restClients.indexWhere((e) => e.name == currentTag);
-        if (sameTagIndex == -1) {
-          restClients.add(
-            UniversalRestClient(
-              name: currentTag,
-              requests: [request],
-              imports: SplayTreeSet<String>.of(imports),
-            ),
+        if (key != _serversVar) {
```
Comment (walsha2) [inline swagger_parser/lib/src/parser/parser.dart:327]: This part resolves #32

**Comment by Carapacik** [comment]
Thanks, everything is done well, I will make some changes to your PR a little later.

**Review by Carapacik — APPROVED** [review]

**Pull request merged**